src/components/Layout/Layout.module.scss

```diff
@@ -8,9 +8,11 @@
   display: flex;
   justify-content: center;
   align-items: center;
+  flex-wrap: wrap;
+  margin-top: -10px;
```

Comment:
Why do we need to assign this negative margin?

Reply:
There was more gap between header and navbar due to margin of the logo.
Shall I proceed and remove margin-top?